packages/common/src/HardhatConfig.js

```diff
@@ -62,7 +66,7 @@ function getHardhatConfig(configOverrides, workingDir = "./", includeTruffle = t
         ovm: true,
         // We use custom logic to only compile contracts within the listed directories, as opposed to choosing which
         // ones to ignore, because there are more contracts to ignore than to include.
-        compileWhitelist: ["oracle/implementation/Finder.sol"],
+        compileWhitelist: ["oracle/implementation/Finder.sol", "insured-bridge/implementation/L2_BridgeDepositBox.sol"],
```

Comment:
Should we remove `Finder` from the whitelist now since the Finder contract require 0.8?

Reply:
ye, can do.